exercises/src/components/exercise/question.jsx

```diff
@@ -80,24 +80,32 @@ class Question extends React.Component {
 
   render() {
     const {
-      question, removeQuestion, moveQuestion, canMoveLeft, canMoveRight,
+      question,
     } = this.props;
-
-
     const { validity } = question;
-    const answers = [];
 
     return (
       <div className="question">
 
         {question.exercise.isMultiPart && this.renderControls()}
 
         {!validity.valid && (
-           <Alert variant="warning">
-             To save your work, you must fill out the {validity.part}
-           </Alert>)}
+          <Alert variant="warning">
+            To save your work, you must fill out the {validity.part}
+          </Alert>)}
 
         <QuestionFormatType question={question} />
+        {!question.exercise.isMultiPart &&
+          (<div className="questio-stimulus">
```

Comment:
I can't tell from just the diff, is this classname declared somewhere (typo?)

Reply:
pretty sure it's a typo.  I don't see it used anywhere.  I'll update it, thanks for noticing!